README.md

```diff
@@ -1 +1,44 @@
-# bodacc
\ No newline at end of file
+# Bodacc
+
+Bodacc est un scrapper qui insère toutes les annonces du Bodacc (2008-2018) en base de données [Bodacc](https://echanges.dila.gouv.fr/OPENDATA/BODACC/)
```

Comment:
Est-ce qu'on peut écrire le README en anglais ? On a pris le pli d'écrire tout ce qui est dans nos repos en anglais, que ça soit les README, les commentaires, les commits.

Les issues, PRs et commentaires sur GitHub qu'on se fait entre nous peuvent rester en français. La seule exception est le titre des PRs car au moment de merger on utilise la stratégie "Squash and Merge" qui transforme une PR en un commit.